lib/topological_inventory/event_listener.rb

```diff
@@ -13,6 +13,9 @@ def process_event(event)
       event.payload['task_id'] = event.payload.delete('id')
       topic = OpenStruct.new(:payload => event.payload, :message => event.message)
       Catalog::DetermineTaskRelevancy.new(topic).process
+    rescue
+      order_item = OrderItem.find_by(:topology_task_ref => event.payload['task_id'].to_s)
+      order_item&.mark_failed("Internal Error. Check log.")
```

Comment:
Similar here as my above comment, except the message should likely be a little bit different since it's failing from a different class. It doesn't look like we need to log anything here though since `DetermineTaskRelevancy` already logs stuff.